packages/@aws-cdk/aws-cognito/lib/user-pool-client.ts

```diff
@@ -56,6 +56,12 @@ export interface OAuthSettings {
    */
   readonly callbackUrls?: string[];
 
+  /**
+   * List of allowed logout URLs for the identity providers.
+   * @default - no logout URLs
+   */
+  readonly logoutUrls?: string[];
```

Comment:
Any reason you've added it within `OAuthSettings`? AFAICT from the documentation, this isn't related to OAuth.

Reply:
Callback url and Logout urls are part of the oAuth configuration. 
 
the settings in the UI is under `App client settings` (title "What identity providers and OAuth 2.0 settings should be used for your app clients".

Reply:
worth mention that CFN user pool clients is flat. but here in CDK I see a structure by `oAuth`. 
anyhow, logoutUrls should be besides callbarckIUrls that currently inside the `oAuth` object.